pandas/core/strings.py

```diff
@@ -53,6 +53,27 @@ def cat_core(list_of_columns, sep):
     return np.sum(list_with_sep, axis=0)
 
 
+def cat_safe(list_of_columns, sep):
```

Comment:
can you type the args  & return value & add a Parameters / Returns section

Reply:
What's your expectation for typing the args? Just `List`? It would strictly speaking be `List[np.array]`, but AFAICT, mypy resp. the typing module doesn't yet support numpy stubs natively.

Reply:
yes that would be fine